src/templates/tutorial.html

```diff
@@ -0,0 +1,44 @@
+<div class="tutorial-container">
+  {% for i in range(1, 4) %}
+    <div class="slide {% if i > 1 %} hidden {% endif %}">
+      <div class="slide-section-top">
+        <span>{{ _('Press') }}</span>
+
+        {% if i == 1 %}
+          <span class="circle ok-key">{{ _('OK') }}</span>
+          <span>{{ _('key to open and use an app') }}</span>
+        {% elif i == 2 %}
+          <span class="circle option-key">{{ _('OPTION') }}</span>
+          <span>{{ _('to add an app to the Apps folder') }}</span>
+        {% elif i == 3 %}
+          <span class="circle apps-key">{{ _('APPS') }}</span>
+          <span>{{ _('to see apps added to the Apps folder') }}</span>
+        {% endif %}
+      </div>
+
+      <div class="slide-section-mid">
+        <div class="slide-image slide-image-0{{ i }}"></div>
+      </div>
+
+      <div class="slide-section-bottom">
+        {% if i > 1 %}
+          <button class="smart-button slide-button focusable"
+                  type="rounded-corner-text"
+                  data-behavior="prev">{{ _('Back') }}</button>
+        {% endif %}
+
+        {% if i < 3 %}
+          <button class="smart-button primary slide-button focusable"
+                  type="rounded-corner-text"
+                  data-behavior="next">{{ _('Next') }}</button>
+        {% endif %}
+
+        {% if i == 3 %}
+          <button class="smart-button primary slide-button focusable"
+                  type="rounded-corner-text"
```

Comment:
The browser will read this as a submit button since this type is not technically valid.